test/functional/page_objects/dashboard_page.ts

```diff
@@ -405,6 +406,16 @@ export class DashboardPageObject extends FtrService {
     await this.waitForRenderComplete();
   }
 
+  public async selectIndexPattern(
```

Comment:
It is misleading to put `selectIndexPattern` into dashboard page object because dashboard application does not have an index pattern select. The The test using `selectIndexPattern` is in lens editor when calling `selectIndexPattern`.

Please remove `selectIndexPattern` to from dashboard page object and call `dataViews.switchTo` directly from `x-pack/test/functional/apps/dashboard/group2/sync_colors.ts`.

Reply:
updated in 2583c89f9f5aff9d394404c16863b375d320e1ea